src/main/reactjs/src/tests/cypress/integration/clerk/translatorOverview/clerk_translator_details.spec.ts

```diff
@@ -34,13 +35,29 @@ describe('ClerkTranslatorOverview:ClerkTranslatorDetails', () => {
     onClerkTranslatorOverviewPage.expectMode(UIMode.EditTranslatorDetails);
   });
 
-  it('should return to view mode when the cancel button is clicked', () => {
+  it('should open a confirmation dialog when cancel is clicked and stay in edit mode if user backs out', () => {
```

Comment:
Voisi lisätä vielä kolmannen testin, että mitään ei edit-modessa tapahdu, jos painaa tallenna-painiketta muuttamatta mitään tietoja (jos tämä siis haluttu toimintamekanismi), näin ainakin toimii paraikaa

Reply:
Hmm.. ei tätä erityistä ole tässä haettu. `PUT` lähtee kyllä backendille ja pyyntö vaikuttaa onnistuvan; bäkkäri ei vaan taida inkrementoida versiota kun ollaan lähetetty samat datat, ja version pysyminen samana aiheuttanee sen että myöskään mitään success-toastia ei tule.

Reply:
Tää ois jollain lailla mielenkiintoista varmaan kyllä hanskata. Ei ehkä tarvitse tehdä kuitenkaan tämän pull requestin yhteydessä korjausta tähän.

Reply:
Onko meillä tarve säilyttää tuo tarkistus `selectedTranslator?.version != prevTranslatorDetails?.version` ClerkTranslatorDetails.tsx:ssä? Tai onko jokin tilanne, jossa success toastia ei pitäisi näyttää vaikka `translatorDetailsStatus === APIResponseStatus.Success` (ja käytössä editmode) ?

Reply:
Otin tarkistuksen pois, tuntuu toimivan konsistentisti ilmankin.